newrelic.js

```diff
@@ -16,7 +16,7 @@ exports.config = {
    */
   license_key: config.newrelic.license_key,
   ssl: true,
-  agent_enabled: true,
+  agent_enabled: config.newrelic.license_key? true : false,
```

Comment:
Identifier 'agent_enabled' is not in camel case.<br>Identifier 'license_key' is not in camel case.